analyzers/its/regression-test.ps1

```diff
@@ -226,6 +226,26 @@ function Show-DiffResults() {
     } -errorMessage $errorMsg
 }
 
+function Invoke-JsonParser()
+{
+    $jsonParser = Join-Path $PSScriptRoot "..\src\ITs.JsonParser\"
```

Comment:
This isn't going to work on CI. You'll need to take the path from packaging folder.